### DISCUSSION TITLE
Installing on Ubuntu - Running as daemon - Issue solved

### DISCUSSION THREAD

**Discussion by MartyOFaherty** [Koenkk/zigbee2mqtt#22104] [General]
My installation failed to start as a service with uninformative messages until I modified /etc/systemd/system/zigbee2mqtt.service from the given example.

I noticed that the line

> User=pi

would be inapplicable to Ubuntu. Having installed Ubuntu from scratch with initial username **marty**, I assumed that I had to change that to

> User=marty

However with some trial and error I found that the correct line is

> User=root

Perhaps it would be worthwhile adding a note clarifying this to the documentation @Koenkk?

**Comment by Nardol** [comment]
Could you give the result of the following?
```shell
ls -l /opt/zigbee2mqtt/config
```

Because if you followed the installation guide, it should not be owned by root.
Except if you did all the steps as root instead of being logged as user `marty`.

In my opinion, running a service as root should be avoided except if really needed.
Maybe having the error you got before could have helped to help you understand what was happening exactly.

**Comment by MartyOFaherty** [comment]
I'll have to get back to you on that. I've put the host in the shed since I got Z2M running as a service, but I didn't enable ssh access to it first. I'll have to bring it back inside and hook up peripherals again. Which probably won't happen now until next weekend.
